docs/command-reference.md

```diff
@@ -0,0 +1,632 @@
+
+### `container network create`
+
+Creates a new network with the given name.
+
+**Usage**
+
+```bash
+container network create NAME
+```
+
+No additional flags; uses global options for debugging and help.
+
+### `container network delete (rm)`
+
+Deletes one or more networks. When deleting multiple networks, pass them as separate arguments. To delete all networks, use `--all`.
+
+**Usage**
+
+```bash
+container network delete [OPTIONS] [NAME...]
+```
+
+**Options**
+
+*   `-a, --all`: delete all defined networks
+*   **Global**: `--debug`, `-h`/`--help`
+
+### `container network list (ls)`
+
+Lists user-defined networks.
+
+**Usage**
+
+```bash
+container network list [OPTIONS]
+```
+
+**Options**
+
+*   `-q, --quiet`: show only network IDs
+*   `--format <format>`: output format (`table` or `json`)
+*   **Global**: `--debug`, `-h`/`--help`
+
+### `container network inspect`
+
+Shows detailed information about one or more networks.
+
+**Usage**
+
+```bash
+container network inspect [OPTIONS] NAME...
+```
+
+Only global flags are available for debugging and help.
+
+## Registry Management
+
+The registry commands manage authentication and defaults for container registries.
+
+### `container registry login`
+
+Authenticates with a registry. Credentials can be provided interactively or via flags. The login is stored for reuse by subsequent commands.
+
+**Usage**
+
+```bash
+container registry login [OPTIONS] SERVER
+```
+
+**Options**
+
+*   `-u, --username <username>`: username for the registry
+*   `--password-stdin`: read the password from STDIN (non-interactive)
+*   `--scheme <scheme>`: registry scheme (`auto`, `http`, `https`)
+*   **Global**: `--debug`, `-h`/`--help`
+
+### `container registry logout`
+
+Logs out of a registry, removing stored credentials.
+
+**Usage**
+
+```bash
+container registry logout [OPTIONS] [SERVER]
+```
+
+No additional flags; uses global options for debugging and help.
+
+### `container registry default` commands
+
+The `registry default` group allows setting, unsetting, and inspecting the default registry used when no registry is specified on image references.
+
+*   `container registry default set [OPTIONS] HOST`: Set the default registry.
+    *   `--scheme <scheme>`: registry scheme (`auto`, `http`, `https`)
+*   `container registry default unset (clear)`: Clears the default registry configuration.
+*   `container registry default inspect`: Displays the current default registry, if any.
+
+## System Management
+
+System commands manage the container apiserver, logs, DNS settings and kernel. These are primarily for macOS hosts.
+
+### `container system start`
+
+Starts the container services and (optionally) installs a default kernel. It will start the `container-apiserver` and background services.
+
+**Usage**
+
+```bash
+container system start [OPTIONS]
+```
+
+**Options**
+
+*   `-p, --path <path>`: path to the `container-apiserver` binary
```

Comment:
I think this option is out of date, as well as the "--kernel-install" option below. We're missing `--app-root` and `--install-root`